packages/augur-tools/src/flash/scripts.ts

```diff
@@ -2116,6 +2119,65 @@ export function addScripts(flash: FlashSession) {
     }
   });
 
+  flash.addScript({
+    name: 'sdk-server',
+    ignoreNetwork: true,
+    options: [
+      {
+        name: 'config-file',
+        abbr: 'f',
+        description: 'SDKConfiguration JSON file',
+      },
+      {
+        name: 'config',
+        abbr: 'c',
+        description: 'serialized SDKConfiguration JSON',
+      },
+    ],
+    async call(this: FlashSession, args: FlashArguments) {
+      const configFile = args.configFile as string;
+      const config = args.config as string;
+
+      let configuration: SDKConfiguration;
+      if (configFile && config) {
```

Comment:
This is so wonderful. We should probably move this up a level to the global configs so flash can take ANY config file.

```flash -c myConfig.json -n MyNetworkEnvironment create-canned-markets```

Does't need to be in this patch. Thoughts? If you agree we can make a ticket.

Reply:
I agree!

https://github.com/AugurProject/augur/issues/6385